app/views/layouts/application.html.erb

```diff
@@ -29,6 +29,7 @@
       <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
     <![endif]-->
 
+    <script src="https://maps.googleapis.com/maps/api/js?libraries=places&language=en&key=AIzaSyDWgc7p4WWFsO3y0MTe50vF4l4NUPcPuwE"></script>
```

Comment:
Well I can then just add it to where it was previously i.e. in `info.html.erb` in the user views. As that's where it is only required. Since I have removed the location_tags.js from the application.js file so it wouldn't cause the previous issue.